Assets/MixedRealityToolkit-SDK/Profiles/DefaultMixedRealityControllerMappingProfile.asset

```diff
@@ -1875,7 +1875,7 @@ MonoBehaviour:
         id: 0
         description: None
         axisConstraint: 0
-      keyCode: 0
+      keyCode: 42
```

Comment:
What's this change?

Reply:
Hmm, not sure. Will double check and revert if ness